support/hololens/ServoApp/strutils.h

```diff
@@ -18,5 +18,5 @@ std::wstring format(const std::wstring &txt, Args... args) {
 }
 
 template <typename... Args> void log(const std::wstring &txt, Args... args) {
-  OutputDebugString((format(txt, args...) + L"\r\n").c_str());
+  std::cout << (format(txt, args...) + L"\r\n").c_str();
```

Comment:
Do we want to use `cout` or would `printf` be better? I don't know C++ that well, but I know some C (and Rust of course!)

Reply:
You'll be missing a `#include` here (iostream iirc).

… and I feel like the redirection doesn't work.

Reply:
Thanks! Just added the include. It's weird how running `./mach build -d` didn't give a warning. (Oh how I miss Rust's module system...)

Reply:
> … and I feel like the redirection doesn't work.

Hmm, what do you mean? I was just implementing what @jdm suggested in #27478

Reply:
I just realized that `./mach build -d` might not have warned me because the header may have been included in a file that already had `#include <iostream>`...

Reply:
`./mach build -d` doesn't build this code.

See https://github.com/servo/servo/wiki/Building-for-UWP

> > … and I feel like the redirection doesn't work.

> Hmm, what do you mean? I was just implementing what @jdm suggested in #27441

I haven't investigated, but this doesn't appear to work.